src/PowerShellEditorServices/Language/AstOperations.cs

```diff
@@ -55,88 +56,95 @@ internal static class AstOperations
+                        "CloneWithNewOffset",
+                        BindingFlags.Instance | BindingFlags.NonPublic);
 #else
-                type.GetMethod(
-                    "CloneWithNewOffset",
-                    BindingFlags.Instance | BindingFlags.NonPublic,
-                    null,
-                    new[] { typeof(int) }, null);
+                    type.GetMethod(
+                        "CloneWithNewOffset",
+                        BindingFlags.Instance | BindingFlags.NonPublic,
+                        null,
+                        new[] { typeof(int) }, null);
 #endif
 
-            IScriptPosition cursorPosition =
-                (IScriptPosition)method.Invoke(
-                    scriptAst.Extent.StartScriptPosition,
-                    new object[] { fileOffset });
-
-            logger.Write(
-                LogLevel.Verbose,
-                string.Format(
-                    "Getting completions at offset {0} (line: {1}, column: {2})",
-                    fileOffset,
-                    cursorPosition.LineNumber,
-                    cursorPosition.ColumnNumber));
-
-            CommandCompletion commandCompletion = null;
-            if (powerShellContext.IsDebuggerStopped)
-            {
-                PSCommand command = new PSCommand();
-                command.AddCommand("TabExpansion2");
-                command.AddParameter("Ast", scriptAst);
-                command.AddParameter("Tokens", currentTokens);
-                command.AddParameter("PositionOfCursor", cursorPosition);
-                command.AddParameter("Options", null);
-
-                PSObject outputObject =
-                    (await powerShellContext.ExecuteCommand<PSObject>(command, false, false))
-                        .FirstOrDefault();
-
-                if (outputObject != null)
+                IScriptPosition cursorPosition =
+                    (IScriptPosition)method.Invoke(
+                        scriptAst.Extent.StartScriptPosition,
+                        new object[] { fileOffset });
+
+                logger.Write(
+                    LogLevel.Verbose,
+                    string.Format(
+                        "Getting completions at offset {0} (line: {1}, column: {2})",
+                        fileOffset,
+                        cursorPosition.LineNumber,
+                        cursorPosition.ColumnNumber));
+
+                if (!powerShellContext.IsAvailable)
                 {
-                    ErrorRecord errorRecord = outputObject.BaseObject as ErrorRecord;
-                    if (errorRecord != null)
-                    {
-                        logger.WriteException(
-                            "Encountered an error while invoking TabExpansion2 in the debugger",
-                            errorRecord.Exception);
-                    }
-                    else
+                    return null;
+                }
+
+                Stopwatch stopwatch = new Stopwatch();
```

Comment:
Minor, but DRY principle:
`var stopwatch = new Stopwatch();`
Also, you could delay initialization and later just call `StopWatch.StartNew()`